Sources/SwifterSwift/SwiftStdlib/RawRepresentableExtensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  RawRepresentableExtensions.swift
+//  SwifterSwift
+//
+//  Created by 方林威 on 2020/7/1.
+//  Copyright © 2020 SwifterSwift
+//
+
+// MARK: - Initializers
+public extension RawRepresentable {
+    
+    /// SwifterSwift: Initializes an  enumeration type. If `rawValue` was malformed, returns `nil`.
+    /// - Parameters:
+    ///   - rawValue: Initialize the value of the enum type..
```

Comment:
```suggestion
    ///   - rawValue: The optional raw value to use for the new instance.
```

Reply:
Yes, I think this description would be better